packages/rum-core/src/domain/rumEventsCollection/resource/resourceCollection.ts

```diff
@@ -127,3 +133,11 @@ function computeRequestTracingInfo(request: RequestCompleteEvent) {
 function computeEntryTracingInfo(entry: RumPerformanceResourceTiming) {
   return entry.traceId ? { _dd: { trace_id: entry.traceId } } : undefined
 }
+
+function toPerformanceEntryRepresentation(entry: RumPerformanceEntry): PerformanceEntryRepresentation {
+  // Safari 10 doesn't support PerformanceEntry
+  if (typeof PerformanceEntry === 'function' && entry instanceof PerformanceEntry) {
```

Comment:
Maybe grouping compatibility check functions like` isPermanceEntryAvailable() `somewhere in `/core` can provide meaningful naming and better reusability?

Reply:
I moved in with similar functions related to "performance" support. I think we shouldn't polute "core" with functions that are only used in "rum". Let me know what you think :)

Reply:
Good for me ;)